tutorials/using_conditional_graphs/README.md

```diff
@@ -0,0 +1,42 @@
+# Using a conditional gate
+
+![image](graph_with_condition_gate.png)
+
+## Overview
+
+Use a condition in an operator as a gate for a downstream operator
+
+## Description
+Basic example with tx and rx operators. Once a condition is met, 
+the mx operator is used to multiply the value from the tx operator, before it is received by a new rx operator.
```

Comment:
Could you please provide an extra sentence or two for motivation? What is a scenario in which this technique might be useful?

Given the scope, would it be better to add this as a maintained HSDK example? (cc @jjomier )